### PR TITLE
hotfix: components parsing

### PR DESCRIPTION
**Pull request by Rapougnac**
follow up of #399 because i've messed up with git

### CODE REVIEW

**Review by abitofevrything — CHANGES_REQUESTED** [review]
Missing version bump in pubspec.yaml (hotfix, so only the last number).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
```
Comment (l7ssha) [inline CHANGELOG.md:1]: new line on top not needed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+## 4.2.1
+__15.11.22__
+- hotfix: fix component deserialization failing when `customId` is `null`
```
Comment (l7ssha) [inline CHANGELOG.md:4]:
```suggestion

- hotfix: fix component deserialization failing when `customId` is `null`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+
+## 4.2.1
+__15.11.22__
```
Comment (l7ssha) [inline CHANGELOG.md:3]:
```suggestion
__15.11.2022__
```

**Comment by l7ssha** [comment]
> Missing version bump in pubspec.yaml (hotfix, so only the last number).

Also missing change of version in constants.dart

**Review by abitofevrything — APPROVED** [review]

**Pull request merged**